q2_feature_classifier/classifier.py

```diff
@@ -182,7 +186,15 @@ def classify_sklearn(reads: DNAIterator, classifier: Pipeline,
                             'complement will cause reads to be reversed '
                             'and complemented prior to classification. '
                             'Default is to autodetect based on the '
-                            'confidence estimates for the first 100 reads.'
+                            'confidence estimates for the first 100 reads.',
+        'chunk_size': 'Number of reads to process in each batch.',
```

Comment:
A bit out of the scope of this PR, but would it make sense to rename this parameter as `number_of_reads` or `reads`.

Reply:
I would be fine with this so long as it made it clear that this parameter was related to chunking the input. Something like `reads_per_batch` or `reads_per_chunk`.